hoot-core/src/main/cpp/hoot/core/OsmMap.cpp

```diff
@@ -933,6 +883,13 @@ bool OsmMap::validate(bool strict) const
 
 void OsmMap::visitRo(ElementVisitor& visitor) const
 {
+  visitNodesRo(visitor);
+  visitWaysRo(visitor);
+  visitRelationsRo(visitor);
+}
+
+void OsmMap::visitNodesRo(ElementVisitor& visitor) const
```

Comment:
I thought for a few minutes that I would do something clever with the ElementType::Type enumeration, but I realized that it's not going to work out. I'm going to leave this as-is for now.